app/views/events/show.html.erb

```diff
@@ -1,7 +1,9 @@
 <div class="event-box">
-  <h2><%=@event.title%></h2>
-  <p><%=@event.description%></p>
-  <p>Start date: <%=@event.start_date.strftime("%A, %B %d, %Y")%> at <%= @event.start_time.strftime("%H:%M") if @event.start_time%></p>
-  <p>End date: <%=@event.end_date.strftime("%A, %B %d, %Y")%> at <%= @event.end_time.strftime("%H:%M") if @event.end_time%> </p>
+    <div id="events_scroll">
```

Comment:
If the changes are mean to be done over `title` and `description` you should create a class on styles for holding the behavior you want on those two elements.

Also notice that you are removing logic on the `<p>` that is not related with the fix of this PR.

Reply:
![screenshot from 2018-02-22 22-31-15](https://user-images.githubusercontent.com/13062617/36559905-52180168-1820-11e8-9ec4-074b39a489ab.png)

Hi @nisevi it looks a little odd when the title and description are separated from start and end date. Tell me what you think

Reply:
Am a little lost here which `<p>` are we talking about?

Reply:
@StlMaris123 the ones with the `if` statement.

Reply:
@StlMaris123 you removed `if @event.end_time` etc. did you mean to do that?